.gitignore

```diff
@@ -8,6 +8,7 @@
 *.egg
 *.egg-info
 dist
+*/setup_base
```

Comment:
Sorry it was getting late for me last night apparently.  We would need the `*/setup_base` symlinks. Not sure what I was thinking before. Updates coming!